yt_dlp/extractor/teamcoco.py

```diff
@@ -76,126 +124,157 @@ class TeamcocoIE(TurnerBaseIE):
         }, {
             'url': 'http://teamcoco.com/israel/conan-hits-the-streets-beaches-of-tel-aviv',
             'only_matching': True,
-        }, {
-            'url': 'https://conan25.teamcoco.com/video/ice-cube-kevin-hart-conan-share-lyft',
-            'only_matching': True,
-        }
+        },
     ]
-    _RECORD_TEMPL = '''id
-        title
-        teaser
-        publishOn
-        thumb {
-          preview
-        }
-        tags {
-          name
-        }
-        duration
-        turnerMediaId
-        turnerMediaAuthToken'''
-
-    def _graphql_call(self, query_template, object_type, object_id):
-        find_object = 'find' + object_type
-        return self._download_json(
-            'https://teamcoco.com/graphql', object_id, data=json.dumps({
-                'query': query_template % (find_object, object_id)
-            }).encode(), headers={
-                'Content-Type': 'application/json',
-            })['data'][find_object]
 
     def _real_extract(self, url):
-        display_id = self._match_id(url)
+        display_id = self._match_id(url).replace('/', '_')
+        webpage = self._download_webpage(url, display_id)
+        data = self._search_nextjs_data(webpage, display_id)['props']['pageProps']['pageData']
+        info = merge_dicts(*[traverse_obj(
+            data, ('blocks', lambda _, v: v['name'] == name, 'props'),
+            get_all=False, expected_type=dict, default={}) for name in ('meta-tags', 'video-player', 'video-info')])
```

Comment:
```suggestion
        info = merge_dicts(*traverse_obj(data, ('blocks',
            lambda _, v: v['name'] in ('meta-tags', 'video-player', 'video-info'),
            'props', {dict}), default={}))
```
(may need re-formatting)